todo_app/data/user_model.py

```diff
@@ -0,0 +1,24 @@
+#Not sure if this is needed? Different method then the suggested one? 
```

Comment:
Agree with you, this looks like an alternative to the User class you defined in `app.py` but it assumes that we're using some sort of ORM (object-relational-mapper - a helper tool for connecting to a DB) - it looks like this isn't used for your code so we should just get rid of this!